JavaScript/JavaScriptSDK/Context/Application.ts

```diff
@@ -1,7 +1,10 @@
-﻿module Microsoft.ApplicationInsights.Context {
+﻿/// <reference path="IApplication.ts" />
+
+module Microsoft.ApplicationInsights.Context {
+
     "use strict";
 
-    export class Application {
+    export class Application implements IApplication {
         /**
```

Comment:
We will need to update the definition files after those interface changes. 

My changes to DefinitelyTyped repo are in review (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9409). 
I would wait for them to approve it and then update it to match new interfaces.